packages/webapp/src/components/Task/TaskComplete/index.jsx

```diff
@@ -26,23 +27,38 @@ export default function PureTaskComplete({
   const COMPLETION_NOTES = 'completion_notes';
   const HAPPINESS = 'happiness';
   const PREFER_NOT_TO_SAY = 'prefer_not_to_say';
-  const COMPLETE_DATE = 'complete_date';
+  const DATE_CHOICE = 'date_choice';
+  const ANOTHER_DATE = 'date_another';
 
   const { t } = useTranslation();
 
+  // Prepare dates
+  const date_due = getDateInputFormat(persistedFormData.due_date);
+  const date_today = getDateInputFormat();
+  const dueDateDisabled = date_due >= date_today;
```

Comment:
@SayakaOno This is disabling the ability to select the first option which I don't think is desired.

Reply:
Removed!

Reply:
@SayakaOno 

I think I was wrong to suggest removing this even though abandon task does it... If you think it is correct I would leave it in.

Reply:
I don't think the option is disabled in AbandonTask. Could you double check?
`/packages/webapp/src/components/Task/AbandonTask/index.jsx`

Reply:
@SayakaOno -- no it doesn't disable it you are right. But it seems like they both _should_ prevent abandoning or completing on a future date. Even though it is not currently in place it seems like that is desired functionality to disable(or hide) the option if the original due date is at some point in the future.